content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -922,9 +932,7 @@ prisma.log.create({
 
 <Admonition type="info">
 
-These _null enums_ do not apply to MongoDB because there the difference between a JSON `null` and a database `NULL` does not exist.
```

Comment:
I'm asking myself what happens when you have a MongoDB database and you try using `Prisma.DbNull`?

Do we accept and just ignore it? Is it a type error? Do we return no results?

Reply:
Without checking, I believe previously we would silently interpret as a string and store a literal "DbNull" string or filter by it. Now it is a type error (and a runtime validation error if you ignore the types or don't use TypeScript).

Reply:
Context: Mongo JSON is different from the other DBs. Json is actually parsed as extJson and transformed to BSON, meaning that DBNull and JsonNull are functionally the same.

Mongo doesn't have the enums on the API level and only accepts pure JSON.

Reply:
As far as I understand this, we don't need a docs change for this, so I'm resolving the conversation.